docs/community/developers.md

```diff
@@ -43,22 +42,21 @@ That will take a LOOONG time the first build, but the layer cache will make it m
 To run Krustlet locally, you can run
 
 ```console
-$ just run-wasi
+$ just run
```

Comment:
This runs the wasCC provider by default.
Is this intented?

Reply:
I figured that whatever's the default for `just run` is probably the right choice for devs. SHould we instead re-write this section to tell them that they should use `just run-wasi` or `just run-wascc` depending on their needs?

Reply:
Note: in the tutorial we actually call this out and inform the user that `run-wasi` is the right choice. Happy to add more clarification here as well. Should be a quick copy n' paste job.